### DISCUSSION TITLE
운영체제 - 페이징

### DISCUSSION THREAD

**Discussion by seongeun42** [Shortudy-10th/tech-for-developer#149] [질의 응답]
페이징 기법 & 알고리즘 관련 질의응답

**Comment by Jihn0118** [comment]
## Q. 페이징 알고리즘 기법 중 LRU가 무엇인가요?

Least Recently Used

최근에 가장 오랫동안 사용하지 않은 페이지를 교체하는 기법입니다.
각 페이지마다 계수기나 스택을 두어 현시점에서 가장 오랫동안 사용하지 않은(가장 오래전에 사용된) 페이지를 교체합니다.

**Comment by zini9188** [comment]
## Q. 페이징 테이블에 대해 설명해주세요
모든 프로세스가 가지고 있는 테이블로 페이지들이 실제로 메인 메모리의 어떤 프레임에 적재되어 있는지를 매핑하는 테이블입니다. MMU를 통해 실제 데이터가 담겨 있는 주소로 변환해줍니다.

**Comment by zini9188** [comment]
### Q. 페이징과 세그멘테이션의 차이는 무엇인가요?
둘의 차이는 프로그램을 분할하는 방식에 있습니다. 페이징의 경우 프로그램을 같은 크기의 페이지로 분할하지만, 세그멘테이션은 논리적인 의미를 기준으로 세그먼트를 분할합니다.
